packages/fast-components-react-base/src/text-field/text-field.props.ts

```diff
@@ -2,19 +2,39 @@ import * as React from "react";
 import { IManagedClasses, ITextFieldClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
 
 export enum TextFieldType {
-   date = "date",
-   email = "email",
-   hidden = "hidden",
-   month = "month",
-   number = "number",
-   password = "password",
-   range = "range",
-   search = "search",
-   tel = "tel",
-   text = "text",
-   time = "time",
-   url = "url",
-   week = "week"
+    /**
+     * @deprecated as of v2.3.0
+     * Will be deleted in version 3.0.
+     */
+    date = "date",
+    email = "email",
+    hidden = "hidden",
```

Comment:
Should we also do hidden? It doesn't seem like hidden is a good fit for this component

Reply:
Yeah - I think we can.